apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/dokument/DokumentRoute.kt

```diff
@@ -25,6 +29,13 @@ fun Route.dokumentRoute(safService: SafService, behandlingKlient: BehandlingKlie
             }
         }
 
+        post("{journalpostId}/ferdigstill") {
+            val request = call.receive<FerdigstillJournalpostRequest>()
+            dokarkivService.ferdigstill(call.parameters["journalpostId"]!!, request)
```

Comment:
Foreslår å lage ein extension function call.notNullParameter(navn), som kastar fornuftig feilmelding viss param er null.

Reply:
Ikke superfan av å lage en extension funksjon for å løse noe kun ett sted, men veldig enig i at den trenger bedre feilmelding. Slenger på en requireNotNull med beskrivende tekst.

Reply:
Einig i det, men har ei hypotese om at det her er noko vi burde hatt ein del plassar - feks

`val journalpostId = call.parameters["journalpostId"]!!
            val dokumentInfoId = call.parameters["dokumentInfoId"]!!`

`val gruppeId = call.request.queryParameters["gruppeId"]!!.toLong()`

`val versjon = call.parameters["versjon"]!!.toLong()`

Reply:
Er nok en del som må ryddes i på et tidspunkt 😅 
Ikke fryktelig kritisk siden det kun er vi som bruker disse API-ene, men greit å få ryddet i det uansett.